tessdata.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: tessdata
+  version: 4.1.0
+  epoch: 0
+  description: Tesseract training data
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - curl-dev
+      - git
+
+data:
+  - name: langs
+    items:
+      afr:
+      ara:
+      aze:
+      bel:
+      ben:
+      bul:
+      cat:
+      ces:
+      chi_sim:
+      chi_tra:
+      chr:
+      dan:
+      deu:
+      eng:
+      enm:
+      epo:
+      equ:
+      est:
+      eus:
+      fin:
+      fra:
+      frk:
+      frm:
+      glg:
+      grc:
+      heb:
+      hin:
+      hrv:
+      hun:
+      ind:
+      isl:
+      ita:
+      ita_old:
+      jpn:
+      kan:
+      kat:
+      khm:
+      kor:
+      lav:
+      lit:
+      mal:
+      mkd:
+      mlt:
+      msa:
+      nld:
+      nor:
+      osd:
+      pol:
+      por:
+      ron:
+      rus:
+      slk:
+      slv:
+      spa:
+      spa_old:
+      sqi:
+      srp:
+      swa:
+      swe:
+      tam:
+      tel:
+      tgl:
+      tha:
+      tur:
+      ukr:
+      vie:
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tesseract-ocr/tessdata
+      tag: ${{package.version}}
+      expected-commit: 4767ea922bcc460e70b87b1d303ebdfed0897da8
+
+subpackages:
+  - name: tesseract-dev
+    pipeline:
+      - uses: split/dev
+
+  - range: langs
+    name: tessdata-${{range.key}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.destdir}}/usr/share/tessdata
+          mv ${{range.key}}.traineddata ${{targets.destdir}}/usr/share/tessdata/
```

Comment:
If we wanna put the content on different subpackages we need to use `${{targets. subpkgdir}}` to point to the subpackage root. `${{targets.destdir}}` is the main package root dir. Here all the subpkg we are creating are empty only .PKGINFO file
```
Package tessdata-dan: Click to expand/collapse
Package tessdata-dan:
Added: /.PKGINFO
```

But do we need to have different subpackage for tessdata. I think as tessdata is a package helping to build the teseract-langs subpackages. Here we can put everything  inside a single package tessdata and in tesseract we can move and put it in a place where we want.

Reply:
I see what you are saying and I am already delivering all tessdata files in a single package to tesseract. There was a a reason I targeted destdir, if you target subpkgdir in tessdata you have to add all those packages in the tesseract.yaml file and that gets large and messy. If you target a destdir and import that into the tesseract yaml file you can simply have one import and use the existing iterative logic to create the correct tesseract sub packages

Reply:
Yes exactly, we do not need sub-packages for tessdata here. we can just have one single package which you have added into the tesseract yaml.

I mean all the tessdata sub-packages like tessdata-dan, tessdata-khm and many others are basically empty with no content in it.

Reply:
I understand what you mean, that is deliberate. Since the range function can only exist within a subpackage heading, we either we create empty subpackages and not use them, targeting the destdir, or have to import every single subpackage in the tesseract file.